packages/server/src/organization/organization-user.entity.ts

```diff
@@ -16,18 +17,14 @@ export class OrganizationUserModel extends BaseEntity {
   @PrimaryGeneratedColumn()
   id: number;
 
-  @Exclude()
   @ManyToOne(
-    type => OrganizationModel,
-    organization => organization.organizationUsers,
+    (type) => OrganizationModel,
+    (organization) => organization.organizationUsers,
```

Comment:
join on organiztionModel on organizationId column and remove the organizationId field

Reply:
I would keep it for easier querying.